tests/datasource/fluent/test_pandas_dbfs_datasource.py

```diff
@@ -0,0 +1,213 @@
+from __future__ import annotations
+
+import logging
+import os
+import pathlib
+import re
+from typing import cast
+
+import boto3
+import botocore
+import pytest
+
+from great_expectations.core.util import DBFSPath
+from great_expectations.datasource.fluent import PandasDBFSDatasource
+from great_expectations.datasource.fluent.data_asset.data_connector import (
+    DBFSDataConnector,
+)
+from great_expectations.datasource.fluent.dynamic_pandas import PANDAS_VERSION
+from great_expectations.datasource.fluent.file_path_data_asset import (
+    _FilePathDataAsset,
+)
+from great_expectations.datasource.fluent.interfaces import TestConnectionError
+from great_expectations.datasource.fluent.pandas_file_path_datasource import (
+    CSVAsset,
+)
+from tests.test_utils import create_files_in_directory
+
+logger = logging.getLogger(__file__)
+
+
+# apply markers to entire test module
+pytestmark = [
+    pytest.mark.skipif(
+        PANDAS_VERSION < 1.2, reason=f"Fluent pandas not supported on {PANDAS_VERSION}"
+    )
+]
+
+
+@pytest.fixture
+def pandas_dbfs_datasource(fs) -> PandasDBFSDatasource:
```

Comment:
Where is this fixture defined? When I try to run this test locally I get a `fixture 'fs' not found` error

Reply:
@Kilo59 Please see `https://github.com/great-expectations/great_expectations/pull/7380#discussion_r1139722918` for the clarification.  Thank you.